code/game/objects/items.dm

```diff
@@ -59,6 +59,7 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	var/armour_penetration = 0 //percentage of armour effectiveness to remove
 	var/list/allowed = null //suit storage stuff.
 	var/obj/item/device/uplink/hidden_uplink = null
+	var/equip_delay = 0 //In deciseconds, how long an item takes to equip; counts only for normal clothing slots, not pockets etc.
 	var/strip_delay = 40
 	var/put_on_delay = 20

```

Comment:
`equip_delay_self` and `equip_delay_other` might be preferable, but then we'd need to change every file with a `put_on_delay`.

Would closely matching the other name, such as with `put_on_self_delay`, help??